### ISSUE TITLE
Recipe: Elastic Stack

### ISSUE THREAD

**Issue by vishal-biyani** [kbrew-dev/kbrew-registry#7]

**Comment by ameydev** [comment]
Is this recipe supposed to install elasticsearch only? or a complete stack of components  like elasticsearch, kibana, logstash etc. ( [helm charts](https://github.com/elastic/helm-charts#charts)  are maintained at [elastic helm repo](https://github.com/elastic/helm-charts). ) ?

**Comment by PrasadG193** [comment]
Our recipe should install complete ELK stack - ELS, Logstash and Kibana https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html

**Comment by PrasadG193** [comment]
@ameydev are you still working on this?

**Comment by ameydev** [comment]
> @ameydev are you still working on this?

yes. I will complete it by today's EOD. Thanks for the clarification.

**Comment by aravindputrevu** [comment]
Happy to help here!

**Comment by ameydev** [comment]
I have already raised a [PR](https://github.com/kbrew-dev/kbrew-registry/pull/27). If it needs any correction then please add it in the review comments.
